### ISSUE TITLE
Create "Principles" documentation

### ISSUE THREAD

**Issue by celestehorgan** [inclusivenaming/org#23]
Document our principles, as a group...

[tbd]

**Comment by LarryKunz** [comment]
I'd like to be involved in documenting the principles.

By way of introduction, I'm an Information Architect at Extreme Networks. We recently started building an inclusive word list in support of the company's Diversity & Inclusion initiative.

**Comment by celestehorgan** [comment]
@LarryKunz Assigned you for now. You won't be the only one on this, just doing it so I don't forget :)

**Comment by celestehorgan** [comment]
(Carla: black & white terms, master/slave terms)

**Comment by mschnoor-intel** [comment]
I'm interested as well.

**Comment by lex-joy** [comment]
Can I volunteer for this one, too? I'd like to work on documenting methods for identifying/investigating problematic terms.

**Comment by joannalee333** [comment]
I'd like to volunteer for this also.  I have a draft list of principles to contribute, and it can be made publicly available

**Comment by celestehorgan** [comment]
@mschnoor-intel @lex-joy @joannalee333 thanks for commenting, all! 

A helpful next step would be gathering resources that are already public so we can take a look at them and see what we do and don't like. If you have anything bookmarked in and around this space of principles, let's start dropping links here :)

**Comment by LarryKunz** [comment]
To get things rolling, here's the word list that we've approved for our Inclusive Language Guideline at Extreme Networks. We have a lot of additional terms under consideration, but they're not yet approved. Our list isn't up on the public web, so I'm posting it here.

Offensive terms: whitelist / blacklist
Inclusive alternatives: permit list / deny list

Offensive terms: white hat / black hat
Inclusive alternatives: ethical / unethical

Offensive terms: master / slave (noun)
Inclusive alternatives: controller / agent or server / client

Offensive terms: master / slave (adj)
Inclusive alternatives: primary / secondary

**Comment by LarryKunz** [comment]
Thank you, Alyssa, for posting this excellent list. To me, the most important factors are found in your last two bullets: 

- We want our terminology to foster a sense of belonging, or inclusion -- the opposite of _otherness_.
- We measure harm or offense from the viewpoint of the members of the group in question, not on the viewpoint (or the intentions) of the ones who publish the terms.

**Community members**: Would you agree with basing our acceptance criteria on Alyssa's last two bullets, perhaps with some light editing. (For example, can we delete "blatantly"?) Then the other three bullets are there to support the criteria, providing real-world guidance to content creators and UX developers. Can anyone recommend anything additional, that I haven't considered here?

**Comment by joannalee333** [comment]
There are some other principles that would also support the high level criteria of fostering a sense of inclusion/opposite of otherness in the Inclusive Prism Guidelines I posted to the shared Google Drive folder and linked to via the Language WS slack channel.  As a suggested process, maybe we can combine Splunk's criteria and the guidelines I posted, and then figure out how to reduce it down to the most essential elements?

**Comment by carlaquinn** [comment]
At IBM, we wrote a few guiding principles that are intended to help people understand and evaluate language. We finally have clearance to share. This is what we've got:
When the terms 

- ‘white’ or ‘black’ are used in the context where white is represented as good or black is represented as bad, this usage reinforces a model that promotes racial bias.
- The use of "master" with the term "slave" in an IT context diminishes the dehumanizing practice of slavery.
- Terms that use color names should be approached with caution to make sure that the colors don't refer to people directly or metaphorically. Do not use color terms that reinforce negative stereotypes.
